src/Resources/config/services.yml

```diff
@@ -13,4 +13,4 @@ services:
 
   HeimrichHannot\FlatpickrBundle\Asset\FrontendAsset:
     calls:
-      - [ setEncoreFrontendAsset, [ '@?HeimrichHannot\EncoreBundle\Asset\FrontendAsset' ] ]
\ No newline at end of file
+      - setEncoreFrontendAsset: [ '@HeimrichHannot\EncoreBundle\Asset\FrontendAsset' ]
```

Comment:
Do you know what the question marks mean? It means that, if you don't have encore it won't throw an exception. Since encore is a loose dependency, the question marks need tto be there. Please revert the change.